### ISSUE TITLE
text-overflow="multiline" IE (edge) vs Chrome

### ISSUE THREAD

**Issue by rmbienek** [BALKANGraph/OrgChartJS#226]
When text-overflow="multiline", Are the y values on these Tspan’s set by the browser or are they calculated by the code?

We have been noticing an odd result. When a text field is changed to text-overflow="multiline", and the font size is increased, IE seems to set the tspan Y value to a much larger value than Chrome. 

Example of the data below. 

“font-size: 14px;” and the text is three line 
Chrome I get Y values of 76, 93, 110 Each 17 apart.
IE (edge) I get Y values of 76, 97.51, 119.02 Each 21.51 apart (diff in line height 4.51).

 “font-size: 18px;” and the text is three line 
Chrome I get Y values of 76, 98, 120 Each 22 apart. 
IE (edge) I get Y values of 76, 121.21, 119.02 Each 36.21 apart. (diff in line height 14.21).

“font-size: 18px;” and the text is three line 
Chrome I get Y values of 76, 103, 130 Each 27 apart. (increase of 5)
IE (edge) I get Y values of 76, 126.91, 177.82 Each, 50.91 apart. (diff in line height 23.91).

**Comment by plamen-peshev** [comment]
Calculated by the code

Are you facing any visual glitch?

**Comment by rmbienek** [comment]
We are seeing the Org cart looks great on chrome using 18px font. But in IE the text exceeds the node walls.

![Org Chart Node](https://user-images.githubusercontent.com/55286886/65771705-6a39d200-e0fe-11e9-861b-9c88c1c98557.jpg)

**Comment by rmbienek** [comment]
Here is another good example. We are using 2 multi line text values. In chrome they look good when the name is less than 2 lines. in IE they overlap.
![Node with 2 Multiline](https://user-images.githubusercontent.com/55286886/65783612-c3613000-e115-11e9-8973-d142d1e1c6c5.jpg)

**Comment by plamen-peshev** [comment]
I have tried to replicate the issue with [this](https://jsfiddle.net/BALKANGraph/h3jm5xsq/?utm_source=website&utm_medium=embed&utm_campaign=h3jm5xsq) demo

It works well for me, what is different in your case?

**Comment by rmbienek** [comment]
Thank you for looking into this issue. Turns out that if you add and amount of “line-height:” to the body section in the CSS, it will blow out the text height in the node. We will be modifying the CSS to eliminate the issue.
However, if you would like to see the results, link below.
https://rmbienek.github.io/
Fiddler did not seem to work with IE so I uploaded your example to the link above with “line-height: 15px;” in the body. If you open it with IE and vs Chrome. There is a significant amount of space added per line in IE.
